packages/react-composites/src/composites/common/AvatarPersona.tsx

```diff
@@ -3,6 +3,8 @@
 
 import { IPersonaProps, Persona, PersonaInitialsColor } from '@fluentui/react';
 import React, { useEffect, useState } from 'react';
+/* @conditional-compile-remove(raise-hand) */
```

Comment:
You can remove conditional compilation for raise hand feature since we have successfully GA's this feature but otherwise looks good!